### ISSUE TITLE
Color code inlets/outlets in "Workflow Editor"

### ISSUE THREAD

**Issue by d0rg0ld** [DM2E/dm2e-gui#20]
A simple color coding for inlets expecting xslt, inlets expecting (legacy)xml, inlets and outlets expecting/delivering RDF could dramatically increase the intuitiveness of the UI

**Comment by kba** [comment]
Agreed, improving the GUI using color coding and other visual feedback is exactly the purpose of the typing system. Very much want to implement this ASAP but need to fix functional bugs and quirks first.

**Comment by kba** [comment]
Though not directly addressing this issue (we're still not making use of the type system in the UI), the new visualization feature makes it easier to understand a workflow wiring. Click on 'Visualize' in workflow editor once https://github.com/DM2E/dm2e-gui/commit/4f65fef1e4a8238aae5616d2cd051a6f61c0a255 gets merged and deployed.
